tests/jerry/array-prototype-sort.js

```diff
@@ -43,9 +43,20 @@ function f(arg1, arg2) {
 	}
 }
 
-var sorted = array.sort(f);
+array.sort(f);
 for (i = 0; i < array.length; i++) {
-	assert(sorted[sorted.length - i - 1] === i);
+	assert(array[array.length - i - 1] === i);
+}
+
+// Sorting sparse array
+var array = [1,,2,,3,,4,undefined,5];
+var expected = [1,2,3,4,5,,,,];
```

Comment:
The following test case print `8`:
```
var expected = [1,2,3,4,5,,,,];

print (expected.length);
```

By the way, assertion at line 57 is incorrect, as it performs assignment, instead of comparison.